.github/workflows/atomistics-compat.yml

```diff
@@ -13,10 +13,9 @@ on:
 jobs:
   build:
     if: |
-      ${{ github.event.type == 'PushEvent' ||
-          ( github.event.type == 'PullRequestEvent'  && contains(github.event.pull_request.labels.*.name, 'integration' ))
-       }}
-
+      github.event_name == 'push' ||
```

Comment:
Doesn't this `or` condition cause the integration tests to run on all the pushes to the master branch?

Reply:
Yes, and I think this should be the intended behavior. In general, we only push the merge commit to the master branch. And every time the master is changed I would run the full test suite with all integration tests to validate the new state. If someone did not use the `integration` label and a merge to the master is failing on the integration tests we are directly alerted.